src/data/PATCHES/patches.ts

```diff
@@ -70,6 +70,7 @@ export const PATCHES = ensureRecord<PatchInfo>()({
 	'7.05': {
 		date: {
 			[GameEdition.GLOBAL]: 1722333600, // 30/07/24 10:00:00 GMT
+			[GameEdition.KOREAN]: 1733212800, // 03/12/24 08:00:00 GMT
```

Comment:
Is the KR edition launching with the 7.05 job balance changes already in place? If not, this date should go under the 7.0 patch definition instead.

Reply:
Yes, KR 7.0 will be released with the 7.05 job balance.